File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/rowversion/dto/GetNextRowVersionResponse.java
package com.rolandvitezhu.todocloud.network.api.rowversion.dto;

import com.google.gson.annotations.SerializedName;
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse;

public class GetNextRowVersionResponse extends BaseResponse {

  @SerializedName("next_row_version")
  public int nextRowVersion;

  public int getNextRowVersion() {
    return nextRowVersion;
  }

  public void setNextRowVersion(int nextRowVersion) {
    this.nextRowVersion = nextRowVersion;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/rowversion/dto/GetNextRowVersionResponse.kt
package com.rolandvitezhu.todocloud.network.api.rowversion.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse
import kotlinx.android.parcel.Parcelize

@Parcelize
data class GetNextRowVersionResponse (
        @SerializedName("next_row_version")
        var nextRowVersion: Int?
) : Parcelable, BaseResponse() {
    constructor() : this(
            null
    )
}